tests/components/poolstation/test_config_flow.py

```diff
@@ -0,0 +1,117 @@
+"""Test the Poolstation config flow."""
+from unittest.mock import AsyncMock, MagicMock, patch
+
+from aiohttp import ClientResponseError
+from pypoolstation import AuthenticationException
+
+from homeassistant import config_entries
+from homeassistant.components.poolstation.const import DOMAIN, TOKEN
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+
+TEST_EMAIL = "bob@burgers.com"
```

Comment:
burgers.com is registered by someone; it's better to use `bob@example.com` even for testing purposes.

Reply:
Fair enough. It was a good pun tho.